app/uk/gov/hmrc/helptosave/actors/UCThresholdManager.scala

```diff
@@ -88,79 +67,31 @@ class UCThresholdManager(thresholdConnectorProxy: ActorRef,
       getValueFromDES() pipeTo self
 
     case GetDESThresholdValueResponse(result) ⇒
```

Comment:
do we need to have both cases `GetDESThresholdValue` and `GetDESThresholdValueResponse` cant we merge both into one case ? 
also whats the benefit of `context become ready(value)` line at the moment ?